crates/rattler/src/install/mod.rs

```diff
@@ -737,19 +760,20 @@ pub fn link_package_sync(
         // files that are either in the clobber map or contain a placeholder,
         // we defer to the regular linking that comes after this block
         // and re-add them to the paths_by_directory map
-        for file in files {
-            if clobber_paths.contains_key(&file.1) || file.0.prefix_placeholder.is_some() {
```

Comment:
The problem was that we were using the computed path here, which already contained the "clobber"-modifications. And thus, we never re-added the path here...